roles/workstation/files/cv.sudoers

```diff
@@ -0,0 +1 @@
+%Local\ Admins ALL=(ALL) ALL
```

Comment:
The idea behind having these owned by the roles was that they were self contained.  While the variable solution more accurately captures state, there are components of the access and configuration model that are not compatible with that design.  It is my opinion that this should stay as is for the time being.